packages/contracts/test/scripts/MigrateMetapool.t.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "src/dollar/core/UbiquityDollarManager.sol";
+import "src/dollar/Staking.sol";
+import "src/dollar/mocks/MockBondingShareV2.sol";
+import "src/dollar/interfaces/IMetaPool.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "forge-std/Test.sol";
+
+contract MetapoolMigrate is Test {
+    address admin = 0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd;
+    UbiquityDollarManager manager =
+        UbiquityDollarManager(0x4DA97a8b831C345dBe6d16FF7432DF2b7b776d98);
+    Staking staking =
+        Staking(payable(0xC251eCD9f1bD5230823F9A0F99a44A87Ddd4CA38));
+    BondingShareV2 bonds =
```

Comment:
```suggestion
    StakingToken stakingToken =
```

Reply:
> I found that MetapoolMigrate test was not part of the CI test so I reran the CI wrokflow cos it was failing locally and now guess what it includes the MetapoolMigrate failing test thinking

When I wrote the script initially the treasury was holding Curve V2 Metapool LP tokens so I migrated those as well. It no longer does so I've removed that part from the scripts and the tests.

Reply:
This is specifically cause of 
`BondingShareV2.Bond memory bond = bonds.getBond(tokens[0]);`
The test is comparing How many LP tokens a user would get on withdrawal pre pool migration versus post. There is no `Struct Bond` in the `StakingShare` contract so I wouldn't be able to use in order to get the this info.

Reply:
Oh I see because its already deployed on mainnet so we can't change. Okay! We will probably need to include a cspell ignore property.